### PR TITLE
Allow multiple row factors in Pivot Table dialog

### PR DESCRIPTION
**Pull request by Vitalis95**
@mmulahvictor and I fixed this issues.
This fixes the improvement issues: row variable be changed to multiple receiver and OK be enabled in the dialogue without having to input either the rows or columns raised on #6634 
@rdstern could you have a look at it

### CODE REVIEW

**Review by rdstern — CHANGES_REQUESTED** [review]
One very minor point.  Ok is now enabled even when there is no data frame in the file.  
There does not need to be a variable, but there should be a current data frame, before Ok is enabled.

The multiple row factors works fine.  Nice!

**Comment by Vitalis95** [comment]
@rdstern  please have a look at it again

**Review by rdstern — APPROVED** [review]
Great

File: instat/dlgPivotTable.vb [hunk 1]
```diff
@@ -110,24 +111,16 @@ Public Class dlgPivotTable
...
-            If ucrChkSelectedVariable.Checked Then
-                If Not ucrReceiverSelectedVariable.IsEmpty Then
-                    ucrBase.OKEnabled(True)
-                Else
-                    ucrBase.OKEnabled(False)
-                End If
-            Else
-                ucrBase.OKEnabled(True)
-            End If
+        If ucrSavePivot.IsComplete AndAlso ucrSelectorPivot.ucrAvailableDataFrames.cboAvailableDataFrames.Text <> "" Then
```
Comment (lloyddewit) [inline instat/dlgPivotTable.vb:122]: Is this safer?
```suggestion
        If ucrSavePivot.IsComplete AndAlso Not String.IsNullOrEmpty(ucrSelectorPivot.ucrAvailableDataFrames.cboAvailableDataFrames.Text) Then
```

File: instat/dlgPivotTable.vb [hunk 2]
```diff
@@ -162,40 +155,47 @@ Public Class dlgPivotTable
...
-    Private Sub ReceiversChanged(ucrChangedControls As ucrCore) Handles ucrReceiverInitialColumnFactor.ControlValueChanged,
-            ucrReceiverInitialRowFactor.ControlValueChanged, ucrReceiverSelectedVariable.ControlValueChanged
+    Private Sub lblInitialColumnFactor_Click(sender As Object, e As EventArgs) Handles lblInitialColumnFactor.Click
+
+    End Sub
+
+    Private Sub ucrReceiverSelectedVariable_Load(sender As Object, e As EventArgs) Handles ucrReceiverSelectedVariable.Load
+
+    End Sub
+
```
Comment (lloyddewit) [inline instat/dlgPivotTable.vb:165]: Remove?
```suggestion
```

**Review by lloyddewit — CHANGES_REQUESTED** [review]
@Vitalis95 This looks really good, just a couple of small suggestions

**Review by lloyddewit — APPROVED** [review]

**Comment by lloyddewit** [comment]
@rdstern Just a couple of small changes since your last approval that should not have affected functionality. Please could you approve again? thanks

**Review by rdstern — APPROVED** [review]

**Pull request merged**